pkg/engine/mutation.go

```diff
@@ -101,7 +101,7 @@ func Mutate(policyContext *PolicyContext) (resp *response.EngineResponse) {
 		}
 
 		// operate on the copy of the conditions, as we perform variable substitution
-		copyConditions, err := transformConditions(rule.AnyAllConditions)
+		copyConditions, err := transformConditions(ruleCopy.AnyAllConditions)
```

Comment:
It seems like @kacejot changed this here: https://github.com/kyverno/kyverno/pull/2216

Can you confirm that it should be `ruleCopy` instead?